Tasks/PowerShellV2/powershell.ts

```diff
@@ -42,6 +51,22 @@ async function run() {
             }
 
             input_arguments = tl.getInput('arguments') || '';
+
+            if (input_arguments) {
+                const [argsArray, telemetry] = parsePowerShellArguments(input_arguments)
+
+                if (featureFlags.enableSecureArgs) {
+                    const json = JSON.stringify(argsArray, null, 2)
+
+                    tl.debug("Writing arguments to temp $argsFile file...")
+                    fs.writeFileSync(argsFile, json)
+                }
+
+                if (featureFlags.enableTelemetry) {
+                    tl.debug("Publishing task telemetry...")
+                    emitTelemetry('TaskHub', 'PowerShellV2', telemetry)
```

Comment:
>emitTelemetry [](http://example.com/codeflow?start=20&length=13)

it might be helpful to emit telemetry even if !input_arguments

Reply:
I'm not sure if it's necessary, because the only thing we will collect is that telemetry is disabled? I think, can just guess it by the `<overall runs> - <with telemetry>` difference